test/span.js

```diff
@@ -123,7 +123,7 @@ describe('span should', () => {
     it('add logs with timestamp, and event', () => {
         let timestamp = new Date(2016, 8, 12).getTime();
         let event = 'some messgae';
-        span.log({ timestamp, event });
+        span._log({ timestamp, event });
```

Comment:
I don't follow - what are all these _ methods, why do we need them, and why would tests run against them instead of the public API?